medusa/providers/torrent/html/gftracker.py

```diff
@@ -175,14 +174,15 @@ def process_column_header(td):
 
                     torrent_size = cells[labels.index('Size/Snatched')].get_text(strip=True).split('/', 1)[0]
                     size = convert_size(torrent_size, units=units) or -1
-
+                    pubdate = cells[labels.index('Added')].get_text(" ")
+                    
```

Comment:
It's `2016-11-25 00:00:00`, absence of timezone means its utc 0 afaik? It's one less datetime import and usage..